networkx/algorithms/traversal/breadth_first_search.py

```diff
@@ -96,7 +95,7 @@ def generic_bfs_edges(G, source, neighbors=None, depth_limit=None, sort_neighbor
     """
     if neighbors is None:
         neighbors = G.neighbors
-    if sort_neighbors is not None:
+    if sort_neighbors is not None and callable(sort_neighbors):
```

Comment:
I would prefer to remove thew `callable` check on the input here. My thinking is that I want to handle this argument whether it is callable or not. The strange case where someone puts in something other than None but it is not callable would silently be ignored with the `callable` check here. I'd rather ignore callable in the check and let the code raise when it is run for uncallable inputs.